src/AboutPage/About.tsx

```diff
@@ -1,8 +1,28 @@
-import React from 'react'
-
 function About() {
   return (
-    <div>About</div>
+    <>
+      <section className='flex justify-center items-center p-10 flex-col'>
+        <h1 className=' text-5xl m-5 md:m-10'>About</h1>
+        <img src="/src/images/hero-img.png" className='w-screen'/>
+      </section>
+
+      <section className='flex flex-col justify-center items-center'>
+        <article className='flex flex-col md:flex-row w-5/6 mx-20 my-5'>
```

Comment:
Need to set a gap on these so there's more space between the columns as shown in the design. You can remove the `pr` on the child element and just set the gap here.